content/blog/2020-11-25-november-20-community-gems.md

```diff
@@ -0,0 +1,195 @@
+tget the locations of the full directory contents.
+
+Why use the API? It's a good way to access DVC-tracked files without downloading
+them to your workspace! No downloads means nothing in your cache. Read more
+about our
+[Python API and the `get_url` function](https://dvc.org/doc/api-reference/get_url#dvcapiget_url)
+in our docs.
+
+### [Q: Can I have more than one `dvc.yaml` file in my project?](https://discord.com/channels/485586884165107732/563406153334128681/777946398250893333)
+
+_Sort of._ Here's what we mean.
+
+By default, DVC pipelines records all your stages (and their inputs and outputs)
+in a single file, `dvc.yaml`. Per directory, you can have one `dvc.yaml` file.
+If you want to run pipelines in a different folder than your project root, you
+could create another `dvc.yaml` in a subdirectory.
+
+However, `dvc.yaml` is intended to be the only file you need to record and
+reproduce pipelines per directory. Pipelines are designed to have all stages
+stored in the same place, and there's currently no method to rename `dvc.yaml`.
+
+If you really, really, _really_ want to put different stages in different files,
+there is one workaround: you can use our hidden option,
+`dvc run --single-stage`, to do this. Your stage will be stored in its own
+`.dvc` file, which can be named whatever you choose. This is a remnant from the
+pre-DVC 1.0 era. But we hope you'll embrace 1.0 and the simplicity of
+`dvc.yaml`!
+
+### [Q: How can I untrack a file that's being tracked by DVC? I want to remove it from remote storage and my local cache, too.](https://discord.com/channels/485586884165107732/563406153334128681/773277514717462548)
+
+We don't have a specific command for this, but you can do it manually:
+
+1. Find the file using its hash from the corresponding `.dvc` file (or, if it's
+   part of a pipeline, the `dvc.lock` file).
+2. Look in your remote storage and remove the file matching the hash.
+3. Look in `.dvc/cache` and remove the file as well.
+
+Your DVC remote storage and cache are simply storage locations, so once your
+file is gone from there it's gone for good.
+
+### [Q: My DVC cache is getting a bit big. Can I clean it?](https://discord.com/channels/485586884165107732/563406153334128681/771275051382341674)
+
+Definitely. Have you seen the command `dvc gc`? It helps you clean your local
+cache- [read up here](https://dvc.org/doc/command-reference/gc). This function
+lets you get granular about what you're keeping; for example, you can instruct
+`dvc gc` to preserve cache files that are currently used your local worksapce,
+tips of Git branches, tagged Git commits or all Git commits. Everything else
+will be removed.
+
+One word of caution: make sure that when you collect garbage from your cache,
+you don't delete any files that you haven't yet pushed to a remote. If this
+happens, you'll delete them permanently. To be safe, it never hurts to
+`dvc push` before cleaning.
```

Comment:
`dvc push` with appropriate options ? ... (by default it will push only current workspace)

Reply:
yeah, added "`dvc push` your files of interest"